src/pages/FormPage.spec.js

```diff
@@ -69,14 +81,16 @@ Magnesium`
 
     render(
       <FormPage
-        onSubmit={jest.fn()}
+        onSubmit={handleSubmit}
         onNavigate={jest.fn()}
         setActivePage={jest.fn()}
+        medicationToEdit={{}}
+        setMedicationToEdit={jest.fn()}
       />
     )
     const form = screen.getByRole('form')
 
-    const button = screen.getByRole('button', { name: 'erstellen' })
+    const button = screen.getByRole('button', { name: 'speichern' })
```

Comment:
Maybe remove the button name from the test, since you can't pass the child as a prop and you might want to change it later like you did before from "erstellen" to "speichern"

Reply:
would be good but I need to be specific because I want to check if this button is disabled and since there are two buttons and I don't want to go with e.g. `buttons[0]` I think this is the way. If you have other recommendations I am happy to hear them!